src/node/chainstate.cpp

```diff
@@ -143,7 +143,7 @@ std::optional<ChainstateLoadVerifyError> VerifyLoadedChainstate(ChainstateManage
         for (CChainState* chainstate : chainman.GetAll()) {
             if (!is_coinsview_empty(chainstate)) {
                 const CBlockIndex* tip = chainstate->m_chain.Tip();
-                if (tip && tip->nTime > GetTime() + MAX_FUTURE_BLOCK_TIME) {
+                if (tip && tip->nTime > get_unix_time_seconds() + 2 * 60 * 60) {
```

Comment:
As there is no need for precision here, wouldn't it be simpler to pass in the time here instead of a function that returns the time?

Reply:
why is this removing `MAX_FUTURE_BLOCK_TIME`?

This reverts https://github.com/bitcoin/bitcoin/commit/26a1147ce56083d7aa820ac115c16b01e47d911c

Reply:
Also, agree with laanwj. This only calls `is_coinsview_empty`, which shouldn't take more than a second (aka the precision of our clock), so it might be better to just pass in the time.